schema/config-linux.json

```diff
@@ -209,6 +209,12 @@
                             "$ref": "defs-linux.json#/definitions/SeccompFlag"
                         }
                     },
+                    "listenerPath": {
+                        "type": "string"
+                    },
+                    "listenerMetadata": {
+                        "type": "string"
+                    },
```

Comment:
same here, I'd move this outside the seccomp section so it can be used also without seccomp